src/templates/next/components/CollectionSort/CollectionSort.tsx

```diff
@@ -0,0 +1,93 @@
+import { CollectionSortProps } from "~/common"
+import { Paragraph } from "../../typography/Paragraph"
+import { useState } from "react"
+import { MdKeyboardArrowDown } from "react-icons/md"
+import { SortDirection, SortKey } from "~/common/CollectionSort"
+import { IoMdCheckmark } from "react-icons/io"
```

Comment:
nit: I think we should standardise to use box icons, which is what is used in OGPDS

Reply:
tbh im a bit confused by the usage
https://www.npmjs.com/package/boxicons#using-via-css

doing `npm i` and `import "boxicons"`, it actually works, but has errors in IDE


![Screenshot 2024-03-28 at 1 26 58 PM](https://github.com/isomerpages/isomer-components/assets/77665182/a399c1ba-4ac2-45e2-8027-30c53f4bad33)

Reply:
Ehh haha it's part of the `react-icons` package but specifically inside the `bi` sub-package. The icons should be prefixed with a `Bi`.